cpp/src/Slice/Preprocessor.cpp

```diff
@@ -140,7 +128,7 @@ Slice::Preprocessor::normalizeIncludePath(const string& path)
 namespace
 {
     vector<string>
-    baseArgs(vector<string> args, bool keepComments, const vector<string>& extraArgs, const string& fileName)
```

Comment:
The `vector` parameter always had exactly a length of 1... so now we just pass that 1 string normally.